src/components/ha-dialog.ts

```diff
@@ -21,6 +21,12 @@ export const createCloseHeading = (hass: HomeAssistant, title: string) => html`
 
 @customElement("ha-dialog")
 export class HaDialog extends MwcDialog {
+  protected renderHeading() {
+    return html`<slot name="heading">
+      <h2 id="title" class="mdc-dialog__title">${this.heading}</h2>
```

Comment:
Weird. You set `heading=${true}` in the more info dialog, but here it is used as a title ?

Reply:
Yeah, so you either enter a string in `heading`, the official way of `mwc-dialog`, or you use the slot, this will only be called if `heading` is truthy...

Hack to make us override the header...

Reply:
Can we add a comment somewhere ?

Reply:
```suggestion
      ${super.renderHeading()}
```